tests/automation_framework/data/work_order/workordergetresult_emptyparameter.json

```diff
@@ -0,0 +1,8 @@
+{
+    "jsonrpc": "2.0",
+    "method": "WorkOrderGetResult",
+    "id": 11,
+    "params": {
+
+    }
+}
```

Comment:
This will fail LINT check, press enter after the }. Same for all .json files

Reply:
done